File: src/outlook-add-in/ReVision/src/taskpane/models/suggestRequest.ts
export class SuggestRequest {
  draft: string;
  targetLanguage: string;
  userLanguage: string;
  writingTone: string;

  constructor(draft: string, targetLanguage: string, userLanguage: string, writingTone: string) {
    this.draft = draft;
    this.targetLanguage = targetLanguage;
    this.userLanguage = userLanguage;
    this.writingTone = writingTone;
  }
}

File: src/api/FunCoding.ReVision/FunCoding.ReVision.WebApi/Models/SuggestRequest.cs
namespace FunCoding.ReVision.WebApi.Models;

public class SuggestRequest
{
    public string Draft { get; set; } = string.Empty;
    public string TargetLanguage { get; set; } = string.Empty;
    public string UserLanguage { get; set; } = string.Empty;
    public string WritingTone { get; set; } = string.Empty;
}
